.kitchen.vagrant.yml

```diff
@@ -48,7 +48,7 @@ platforms:
 - name: centos-7
   driver:
     vagrantfiles:
-      - VagrantfileA.rb
+      - /test/Vagrantfile_setup.rb
```

Comment:
Does this work for you? For me I need to remove the leading `/`. This would only work if you had this repo checked out in your root directory...I hope you aren't doing that :)

So for me it looks like `- test/Vagrantfile_setup.rb`